db/migrate/20250207194638_adjust_securities_indexes.rb

```diff
@@ -0,0 +1,6 @@
+class AdjustSecuritiesIndexes < ActiveRecord::Migration[7.2]
+  def change
+    remove_index :securities, name: "index_securities_on_ticker_and_exchange_mic"
+    add_index :securities, [ :ticker, :exchange_operating_mic ], unique: true
+  end
```

Comment:
We should also throw in a NOT NULL constraint on `ticker` here since that's only enforced on the model right now

Reply:
That said, we may want to defer the _new_ constraints until after we've run all the data migrations as I believe this migration will fail on our production data.

We can then introduce this after the data migrations (and follow-ups) are complete